.github/workflows/cd.yaml

```diff
@@ -55,3 +55,34 @@ jobs:
           AWS_REGION: us-east-1
         run: |
           aws s3 cp $BINARY_NAME s3://release.ariga.io/atlas-action/$BINARY_NAME
+
+  # Run end-to-end test on the published binary.
+  e2e-test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ariga/setup-atlas@master
+        with:
+          cloud-token: ${{ secrets.INTEGRATION_ATLAS_CLOUD_TOKEN }}
+      - name: "Run Migrate Lint"
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        uses: jenseng/dynamic-uses@v1
```

Comment:
GitHub action doesn't accept dynamic values it the `uses:` field.
Because I want to download the uploaded binary in build job, I need to use this custom action